torch/fx/experimental/proxy_tensor.py

```diff
@@ -303,6 +304,29 @@ def __torch_dispatch__(cls, func_overload, types, args=(), kwargs=None):
                     break
         assert proxy_mode is not None, "At least one argument must be a ProxyTensor"
 
+        if func_overload.__name__ == 'torch.cond':
```

Comment:
I don't mean to keep expanding the scope of the prototype, but one goal is to be able to define new operators for existing Tensor subclasses. So ideally we would define ProxyTensor's behavior for torch.cond inside the `control_flow.py` file instead of needing to modify the implementation of ProxyTensor

The long-term solution to this is to have some sort of a "custom subclass operator registration API" but the short-term thing we can do here is change cond's implementation for the "python key" to (1) check if the inputs are all ProxyTensor, and if so, (2) call a function that does exactly what we do here.

Reply:
I have no intention of actually having this hack inside proxy tensor, this is just a useful place to put it before it works end to end. Until it works end to end, I'm going to keep it here. It does not work fully correctly yet. We are still unfortunately capturing the book as passed into make_fx. Talking to @Chillee about it.

Reply:
I added a suspend_mode to handle this.